apps/bajour/src/components/briefing/basel-briefing.tsx

```diff
@@ -391,9 +397,23 @@ export const BaselBriefing = ({teasers, blockStyle}: BaselBriefingProps) => {
                 </Author>
               )}
 
-              <ReadMoreButton variant="outlined" color="inherit" size="small">
-                Ganzes Briefing
-              </ReadMoreButton>
+              <ButtonRow>
+                {(blockStyle as BriefingType) === BriefingType.BaselBriefing && (
```

Comment:
could use 

```suggestion
                {isBaselBriefing({blockStyle}) && (
```

instead